src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -155,4 +155,36 @@
 
   </ItemGroup>
 
+  <!-- ******************************************************************************* -->
```

Comment:
The final solution is a bit tricky.

1. We use shell 14 everywhere, and by importing IssueViz into IssueViz.Security, IssueViz.Security also has a "built-in" reference to shell 14, no matter which VS version it's built on.

Here I re-add shell 14, this time with an alias `shell`. I do the same for shell 15.
This allows me to force import `IWpfTableControlProvider` from the right shell assembly by doing:
```
extern alias shell;

using IWpfTableControlProvider = shell::Microsoft.Internal.VisualStudio.Shell.TableControl.IWpfTableControlProvider;
```
... while all the other classes from shell, like `Microsoft.VisualStudio.Shell.PackageRegistrationAttribute`, just get imported from shell 14 via IssueViz project reference. If IssueViz project reference is removed, the compilation fails due to missing Shell dll -- which either needs to be directly imported, or used via `extern alias`.
Hence why I decided to directly import it in IssueViz.Security.csproj, instead of relying on the IssueViz dependency.

2. Unfortunately, shell 15 brings with it `Microsoft.VisualStudio.Shell.Framework`, which re-declares the class `ServiceProgressData` (from package InitializeAsync method), that already exists in `Microsoft.VisualStudio.Shell.Immutable.14.0`. So in order to "silence" this conflict, I add an alias to `Microsoft.VisualStudio.Shell.Framework`. This way it cannot be used without the alias, there is no conflict, and `ServiceProgressData` is taken from Shell.Immutable.